jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -635,7 +635,7 @@ public void succeeded(Connection connection)
         public void failed(Throwable x)
         {
             if (LOG.isDebugEnabled())
-                LOG.debug("Connection creation failed {}", reserved, x);
+                LOG.atDebug().setCause(x).log("Connection creation failed {}", reserved);
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.